test/framework/shoot_utils.go

```diff
@@ -272,6 +272,15 @@ func setShootNetworkingSettings(shoot *gardencorev1beta1.Shoot, cfg *ShootCreati
 		clearDNS = false
 	}
 
+	switch strings.ToLower(cfg.ipFamily) {
+	case "dual-stack", "dualstack":
+		shoot.Spec.Networking.IPFamilies = []gardencorev1beta1.IPFamily{gardencorev1beta1.IPFamilyIPv6, gardencorev1beta1.IPFamilyIPv4}
```

Comment:
would a dualstack configuration with `ipv4` as the primary stack be a valid / test-worthy configuration option as well and should this be configurable?

Reply:
In the shoot specification, it is possible to distinguish between the primary and secondary address family. It has some consequences, e.g. default address family for services, but the impact is limited. From my perspective, it is sufficient to test just one dual-stack case with IPv6 as primary so that IPv6 is more likely to be used. I would leave the `IPv4, IPv6` case untested to limit the amount of resources used for testing.
However, I am open to changing it in case you have the desire for full test coverage.

Reply:
Thanks for for the detailed explanation. From my perspective it would be beneficial to have the configuration option for the `IPv4, IPv6` case available. Then we can decide on the level of the test flavors (testmachinery configuration), which tests we want to run and have the option to switch up test cases also on short notice.

Reply:
Updated the change accordingly to directly expose the IP families field. This way tests can specify all possible combinations if required.

PTAL.